src/main/java/io/vertx/rabbitmq/RabbitMQOptions.java

```diff
@@ -1,21 +1,29 @@
 package io.vertx.rabbitmq;
 
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
 import com.rabbitmq.client.Address;
 import com.rabbitmq.client.ConnectionFactory;
+
 import io.vertx.codegen.annotations.DataObject;
 import io.vertx.core.json.JsonObject;
-
-import java.util.ArrayList;
-import java.util.Collections;
-import java.util.List;
+import io.vertx.core.net.JksOptions;
+import io.vertx.core.net.KeyCertOptions;
+import io.vertx.core.net.NetClientOptions;
+import io.vertx.core.net.PemKeyCertOptions;
+import io.vertx.core.net.PemTrustOptions;
+import io.vertx.core.net.PfxOptions;
+import io.vertx.core.net.TrustOptions;
 
 /**
  * RabbitMQ client options, most
  *
  * @author <a href="mailto:julien@julienviet.com">Julien Viet</a>
  */
 @DataObject(generateConverter = true)
-public class RabbitMQOptions {
+public class RabbitMQOptions extends NetClientOptions{
```

Comment:
I think we should override instead `TCPSSLOptions` and have `SSLHelper` constructor that takes `TCPSSLOptions` be public. That would create less coupling.

Reply:
Also NetClient options defines somehow more than what we need and people might take it for granted.

Reply:
Actually I can see now it can reuse `setReconnectAttempts` and the other for delay. So well it's fine this way